srcpkgs/cross-arm-none-eabi/template

```diff
@@ -1,367 +1,10 @@
-# Template file for 'cross-arm-none-eabi'
-_binutils_version=2.32
-_gcc_version=7.3.0
-_newlib_version=2.4.0.20161025
-
+# Template file for 'cross-${_triplet}'
 _triplet=arm-none-eabi
-_sysroot="/usr/${_triplet}"
-
 pkgname=cross-${_triplet}
-version=0.3
-revision=2
+version=0.4
+revision=1
+build_style=meta
+depends="cross-arm-none-eabi-binutils cross-arm-none-eabi-gcc cross-arm-none-eabi-newlib"
 short_desc="GNU cross bare metal toolchain"
 maintainer="Thomas Bernard <thomas@famillebernardgouriou.fr>"
 homepage="https://www.voidlinux.org/"
```

Comment:
Append `license="metapackage"` here to satisfy xlint. Better yet install `xtools` and run `xlint srcpkg/<yourpkg>/template`.

Reply:
xlint did not complain